Crush when 'Pass Direct Path for external player' is enabled and starting a video with VLC

### Describe the bug

Since the update to v15 whenever I open a media file I get 'Ops something went wrong..." error. 
After some debugging I found out, if I turn off the _**Preferences -> Playback -> Pass Direct Path for external player**_ the error goes away.
 

### Logs

---
client: Jellyfin for Android TV
client_version: 0.15.0-beta.1
client_repository: https://github.com/jellyfin/jellyfin-androidtv
type: crash_report
format: markdown
---
### Logs

***Stack Trace***: 
```log
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.jellyfin.androidtv/org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.List org.jellyfin.sdk.model.api.MediaSourceInfo.getMediaStreams()' on a null object reference
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2914)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3049)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1809)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6680)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.List org.jellyfin.sdk.model.api.MediaSourceInfo.getMediaStreams()' on a null object reference
	at org.jellyfin.androidtv.data.compat.StreamInfo.getSubtitleProfiles(StreamInfo.java:301)
	at org.jellyfin.androidtv.data.compat.StreamInfo.getSubtitleProfiles(StreamInfo.java:291)
	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity.adaptExternalSubtitles(ExternalPlayerActivity.java:445)
	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity.startExternalActivity(ExternalPlayerActivity.java:418)
	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity.launchExternalPlayer(ExternalPlayerActivity.java:309)
	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity.onCreate(ExternalPlayerActivity.java:121)
	at android.app.Activity.performCreate(Activity.java:7144)
	at android.app.Activity.performCreate(Activity.java:7135)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2894)
	... 11 more

```  
***Logcat***: 
```log
--------- beginning of main
11-16 19:38:13.185 I/jellyfin-apiclient(18272): Response received from: http://192.168.0.108:8096/Sessions/Playing
11-16 19:38:13.535 D/ACRA    (18272): onActivityStopped class org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity
11-16 19:38:13.538 D/ACRA    (18272): onActivitySaveInstanceState class org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity
11-16 19:38:13.540 D/ACRA    (18272): onActivityStopped class org.jellyfin.androidtv.ui.browsing.MainActivity
11-16 19:38:13.555 D/ACRA    (18272): onActivitySaveInstanceState class org.jellyfin.androidtv.ui.browsing.MainActivity
11-16 19:38:13.676 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(18272): Receiving (raw) message {"MessageType":"UserDataChanged","MessageId":"52ecad42cf8846e4828103b47935b052","Data":{"UserId":"25fc68de4aca429c9d32dc0f92026ffc","UserDataList":[{"PlaybackPositionTicks":0,"PlayCount":5,"IsFavorite":false,"LastPlayedDate":"2022-11-16T18:38:13.4705577Z","Played":false,"Key":"366972001001","ItemId":"5c7e697f10441fd46a6a336793620cd5"},{"UnplayedItemCount":10,"PlaybackPositionTicks":0,"PlayCount":0,"IsFavorite":false,"Played":false,"Key":"366972001","ItemId":"8aae5c80f6572f182fdbe3c46a983b3c"}]}}
11-16 19:38:13.996 W/System  (18272): A resource failed to call end. 
11-16 19:38:21.628 D/ExternalPlayerActivity(18272): Returned from player, result <0>, extra data <null>
11-16 19:38:21.629 I/jellyfin-apiclient(18272): ReportPlaybackStopped: Item 5c7e697f-1044-1fd4-6a6a-336793620cd5, Ticks: 0
11-16 19:38:21.629 D/jellyfin-apiclient(18272): Adding request to queue: http://192.168.0.108:8096/Sessions/Playing/Stopped
11-16 19:38:21.672 D/ACRA    (18272): onActivityStarted class org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity
11-16 19:38:21.673 I/ProcessLifecycleInitializer$create(18272): Process lifecycle started
11-16 19:38:21.674 D/SessionRepositoryImpl(18272): Restoring session
11-16 19:38:21.680 D/ACRA    (18272): onActivityResumed class org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity
11-16 19:38:21.682 D/BackgroundServiceFragment(18272): Restoring active backgrounds
11-16 19:38:21.682 D/BackgroundServiceFragment(18272): Restoring background drawable
11-16 19:38:21.805 I/jellyfin-apiclient(18272): Response received from: http://192.168.0.108:8096/Sessions/Playing/Stopped
11-16 19:38:22.349 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(18272): Receiving (raw) message {"MessageType":"UserDataChanged","MessageId":"0308d62c47794345abed29c326f79b44","Data":{"UserId":"25fc68de4aca429c9d32dc0f92026ffc","UserDataList":[{"PlaybackPositionTicks":0,"PlayCount":5,"IsFavorite":false,"LastPlayedDate":"2022-11-16T18:38:13.4705577Z","Played":false,"Key":"366972001001","ItemId":"5c7e697f10441fd46a6a336793620cd5"},{"UnplayedItemCount":10,"PlaybackPositionTicks":0,"PlayCount":0,"IsFavorite":false,"Played":false,"Key":"366972001","ItemId":"8aae5c80f6572f182fdbe3c46a983b3c"}]}}
11-16 19:38:23.257 D/OpenGLRenderer(18272): endAllActiveAnimators on 0xd094e900 (RippleDrawable) with handle 0xcc618070
11-16 19:38:23.275 D/ACRA    (18272): onActivityPaused class org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity
11-16 19:38:23.293 D/ACRA    (18272): onActivityStarted class org.jellyfin.androidtv.ui.browsing.MainActivity
11-16 19:38:23.294 I/ViewTarget(18272): Glide treats LayoutParams.WRAP_CONTENT as a request for an image the size of this device's screen dimensions. If you want to load the original image and are ok with the corresponding memory cost and OOMs (depending on the input size), use override(Target.SIZE_ORIGINAL). Otherwise, use LayoutParams.MATCH_PARENT, set layout_width and layout_height to fixed dimension, or use .override() with fixed dimensions.
11-16 19:38:23.295 I/chatty  (18272): uid=10108(org.jellyfin.androidtv) identical 2 lines
11-16 19:38:23.295 I/ViewTarget(18272): Glide treats LayoutParams.WRAP_CONTENT as a request for an image the size of this device's screen dimensions. If you want to load the original image and are ok with the corresponding memory cost and OOMs (depending on the input size), use override(Target.SIZE_ORIGINAL). Otherwise, use LayoutParams.MATCH_PARENT, set layout_width and layout_height to fixed dimension, or use .override() with fixed dimensions.
11-16 19:38:23.297 D/ACRA    (18272): onActivityResumed class org.jellyfin.androidtv.ui.browsing.MainActivity
11-16 19:38:23.300 D/BackgroundServiceFragment(18272): Restoring active backgrounds
11-16 19:38:23.301 D/BackgroundServiceFragment(18272): Restoring background drawable
11-16 19:38:23.621 D/ACRA    (18272): onActivityStopped class org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity
11-16 19:38:23.622 D/ACRA    (18272): onActivityDestroyed class org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity
11-16 19:38:24.053 D/FullDetailsFragment(18272): current time 1668623904052 last playback event time 1668623901630 last refresh time 1668623890674
11-16 19:38:24.081 D/FullDetailsFragment(18272): Updating info after playback
11-16 19:38:24.082 D/jellyfin-apiclient(18272): Adding request to queue: http://192.168.0.108:8096/Users/25fc68de-4aca-429c-9d32-dc0f92026ffc/Items/5c7e697f10441fd46a6a336793620cd5?format=json
11-16 19:38:24.167 I/jellyfin-apiclient(18272): Response received from: http://192.168.0.108:8096/Users/25fc68de-4aca-429c-9d32-dc0f92026ffc/Items/5c7e697f10441fd46a6a336793620cd5?format=json
11-16 19:38:24.609 D/NavigationRepositoryImpl(18272): Navigating back
11-16 19:38:24.660 D/HomeRowsFragment(18272): Refresh item Foundation S1 E1
11-16 19:38:24.661 I/ItemRowAdapter(18272): Re-retrieving row of type NextUp
11-16 19:38:24.662 I/org.jellyfin.sdk.api.client.KtorClient(18272): GET http://192.168.0.108:8096/Users/25fc68de-4aca-429c-9d32-dc0f92026ffc/Items/5c7e697f-1044-1fd4-6a6a-336793620cd5
11-16 19:38:24.662 D/jellyfin-apiclient(18272): Adding request to queue: http://192.168.0.108:8096/Shows/NextUp?Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&UserId=25fc68de-4aca-429c-9d32-dc0f92026ffc&Limit=50&ImageTypeLimit=1&format=json
11-16 19:38:24.663 I/HomeRowsFragment(18272): Updating audio queue in HomeFragment (onResume)
11-16 19:38:24.666 D/NavigationRepositoryImpl(18272): Navigating to Fragment(fragment=class org.jellyfin.androidtv.ui.home.HomeFragment (Kotlin reflection is not available), arguments=Bundle[{}])
11-16 19:38:24.936 I/jellyfin-apiclient(18272): Response received from: http://192.168.0.108:8096/Shows/NextUp?Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&UserId=25fc68de-4aca-429c-9d32-dc0f92026ffc&Limit=50&ImageTypeLimit=1&format=json
11-16 19:38:25.127 D/MediaManager(18272): Removed event listener.  Total listeners: 1
11-16 19:38:25.174 I/lyfin.androidt(18272): Background concurrent copying GC freed 53068(2MB) AllocSpace objects, 10(496KB) LOS objects, 43% free, 15MB/27MB, paused 377us total 235.848ms
11-16 19:38:25.184 W/System  (18272): A resource failed to call end. 
11-16 19:38:25.644 I/lyfin.androidt(18272): NativeAlloc concurrent copying GC freed 11701(687KB) AllocSpace objects, 0(0B) LOS objects, 44% free, 15MB/27MB, paused 298us total 216.895ms
11-16 19:38:26.366 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(18272): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
11-16 19:38:26.373 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(18272): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"8327f2958e67446091ae9864905a7b1a"}
11-16 19:38:27.349 D/NavigationRepositoryImpl(18272): Navigating to Activity(activity=class org.jellyfin.androidtv.ui.preference.PreferencesActivity (Kotlin reflection is not available), extras=Bundle[{screen=org.jellyfin.androidtv.ui.preference.screen.UserPreferencesScreen, screen_args=Bundle[{}]}])
11-16 19:38:27.360 D/NavigationRepositoryImpl$navigate$action(18272): Navigating to nothing
11-16 19:38:27.363 D/ACRA    (18272): onActivityPaused class org.jellyfin.androidtv.ui.browsing.MainActivity
11-16 19:38:27.378 W/ActivityThread(18272): handleWindowVisibility: no activity for token android.os.BinderProxy@e88be5a
11-16 19:38:27.389 D/ACRA    (18272): onActivityCreated class org.jellyfin.androidtv.ui.preference.PreferencesActivity
11-16 19:38:27.390 I/AuthenticatedUserCallbacks(18272): Activity org.jellyfin.androidtv.ui.preference.PreferencesActivity is ignored
11-16 19:38:27.391 I/AppThemeCallbacks(18272): Applying theme: DARK
11-16 19:38:27.405 D/ACRA    (18272): onActivityStarted class org.jellyfin.androidtv.ui.preference.PreferencesActivity
11-16 19:38:27.430 D/ACRA    (18272): onActivityResumed class org.jellyfin.androidtv.ui.preference.PreferencesActivity
11-16 19:38:45.193 D/OpenGLRenderer(18272): endAllActiveAnimators on 0xcaf98e80 (RippleDrawable) with handle 0xcaeb5cb0
11-16 19:38:47.436 D/ACRA    (18272): onActivityPaused class org.jellyfin.androidtv.ui.preference.PreferencesActivity
11-16 19:38:47.467 D/ACRA    (18272): onActivityResumed class org.jellyfin.androidtv.ui.browsing.MainActivity
11-16 19:38:47.470 D/BackgroundServiceFragment(18272): Restoring active backgrounds
11-16 19:38:47.472 D/BackgroundServiceFragment(18272): Restoring background drawable
11-16 19:38:47.473 I/HomeRowsFragment(18272): Updating audio queue in HomeFragment (onResume)
11-16 19:38:47.491 D/ACRA    (18272): onActivityStopped class org.jellyfin.androidtv.ui.preference.PreferencesActivity
11-16 19:38:47.493 D/ACRA    (18272): onActivityDestroyed class org.jellyfin.androidtv.ui.preference.PreferencesActivity
11-16 19:38:49.073 I/lyfin.androidt(18272): NativeAlloc concurrent copying GC freed 38755(1941KB) AllocSpace objects, 2(104KB) LOS objects, 44% free, 15MB/27MB, paused 347us total 223.641ms
11-16 19:38:49.590 D/ItemLauncher(18272): Item selected: 0 - The Emperor’s Peace (Episode)
11-16 19:38:49.591 D/NavigationRepositoryImpl(18272): Navigating to Fragment(fragment=class org.jellyfin.androidtv.ui.itemdetail.FullDetailsFragment (Kotlin reflection is not available), arguments=Bundle[{ItemId=5c7e697f-1044-1fd4-6a6a-336793620cd5}])
11-16 19:38:49.630 D/jellyfin-apiclient(18272): Adding request to queue: http://192.168.0.108:8096/Users/25fc68de-4aca-429c-9d32-dc0f92026ffc/Items/5c7e697f-1044-1fd4-6a6a-336793620cd5?format=json
11-16 19:38:49.639 D/MediaManager(18272): Added event listener.  Total listeners: 2
11-16 19:38:49.654 W/RecyclerView(18272): No adapter attached; skipping layout
11-16 19:38:49.750 I/jellyfin-apiclient(18272): Response received from: http://192.168.0.108:8096/Users/25fc68de-4aca-429c-9d32-dc0f92026ffc/Items/5c7e697f-1044-1fd4-6a6a-336793620cd5?format=json
11-16 19:38:50.024 D/jellyfin-apiclient(18272): Adding request to queue: http://192.168.0.108:8096/Shows/469174e1deff99a69a660a437bd56aa1/Episodes?AdjacentTo=5c7e697f10441fd46a6a336793620cd5&UserId=25fc68de-4aca-429c-9d32-dc0f92026ffc&format=json
11-16 19:38:50.042 D/FullDetailsFragment(18272): Item type: Episode
11-16 19:38:50.043 D/jellyfin-apiclient(18272): Adding request to queue: http://192.168.0.108:8096/Users/25fc68de-4aca-429c-9d32-dc0f92026ffc/Items?ParentId=8aae5c80f6572f182fdbe3c46a983b3c&StartIndex=1&recursive=false&Fields=PrimaryImageAspectRatio%2COverview%2CItemCounts%2CDisplayPreferencesId%2CChildCount&Limit=20&IncludeItemTypes=Episode&format=json
11-16 19:38:50.162 I/ViewTarget(18272): Glide treats LayoutParams.WRAP_CONTENT as a request for an image the size of this device's screen dimensions. If you want to load the original image and are ok with the corresponding memory cost and OOMs (depending on the input size), use override(Target.SIZE_ORIGINAL). Otherwise, use LayoutParams.MATCH_PARENT, set layout_width and layout_height to fixed dimension, or use .override() with fixed dimensions.
11-16 19:38:50.163 I/ViewTarget(18272): Glide treats LayoutParams.WRAP_CONTENT as a request for an image the size of this device's screen dimensions. If you want to load the original image and are ok with the corresponding memory cost and OOMs (depending on the input size), use override(Target.SIZE_ORIGINAL). Otherwise, use LayoutParams.MATCH_PARENT, set layout_width and layout_height to fixed dimension, or use .override() with fixed dimensions.
11-16 19:38:50.184 I/jellyfin-apiclient(18272): Response received from: http://192.168.0.108:8096/Shows/469174e1deff99a69a660a437bd56aa1/Episodes?AdjacentTo=5c7e697f10441fd46a6a336793620cd5&UserId=25fc68de-4aca-429c-9d32-dc0f92026ffc&format=json
11-16 19:38:50.187 I/jellyfin-apiclient(18272): Response received from: http://192.168.0.108:8096/Users/25fc68de-4aca-429c-9d32-dc0f92026ffc/Items?ParentId=8aae5c80f6572f182fdbe3c46a983b3c&StartIndex=1&recursive=false&Fields=PrimaryImageAspectRatio%2COverview%2CItemCounts%2CDisplayPreferencesId%2CChildCount&Limit=20&IncludeItemTypes=Episode&format=json
11-16 19:38:50.193 D/ItemRowAdapterHelperKt(18272): Creating items from 0 existing and 9 new, adapter size is 0
11-16 19:38:50.399 D/FullDetailsFragment(18272): current time 1668623930398 last playback event time 1668623901630 last refresh time 1668623930041
11-16 19:38:50.927 D/jellyfin-apiclient(18272): Adding request to queue: http://192.168.0.108:8096/Shows/469174e1-deff-99a6-9a66-0a437bd56aa1/Episodes?Fields=MediaSources%2CMediaStreams%2CPath%2CChapters%2COverview%2CPrimaryImageAspectRatio%2CChildCount&UserId=25fc68de-4aca-429c-9d32-dc0f92026ffc&SeasonId=8aae5c80-f657-2f18-2fdb-e3c46a983b3c&IsMissing=false&IsVirtualUnaired=false&format=json
11-16 19:38:51.569 I/jellyfin-apiclient(18272): Response received from: http://192.168.0.108:8096/Shows/469174e1-deff-99a6-9a66-0a437bd56aa1/Episodes?Fields=MediaSources%2CMediaStreams%2CPath%2CChapters%2COverview%2CPrimaryImageAspectRatio%2CChildCount&UserId=25fc68de-4aca-429c-9d32-dc0f92026ffc&SeasonId=8aae5c80-f657-2f18-2fdb-e3c46a983b3c&IsMissing=false&IsVirtualUnaired=false&format=json
11-16 19:38:51.747 D/NavigationRepositoryImpl(18272): Navigating to Activity(activity=class org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity (Kotlin reflection is not available), extras=Bundle[{Position=0}])
11-16 19:38:51.761 D/NavigationRepositoryImpl$navigate$action(18272): Navigating to nothing
11-16 19:38:51.762 D/ACRA    (18272): onActivityPaused class org.jellyfin.androidtv.ui.browsing.MainActivity
11-16 19:38:51.773 W/ActivityThread(18272): handleWindowVisibility: no activity for token android.os.BinderProxy@cee3903
11-16 19:38:51.779 D/ACRA    (18272): onActivityCreated class org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity
11-16 19:38:51.779 I/AppThemeCallbacks(18272): Applying theme: DARK
11-16 19:38:51.788 I/BackgroundService(18272): Adding BackgroundServiceFragment to activity
11-16 19:38:51.791 D/AndroidRuntime(18272): Shutting down VM
--------- beginning of crash
11-16 19:38:51.792 E/AndroidRuntime(18272): FATAL EXCEPTION: main
11-16 19:38:51.792 E/AndroidRuntime(18272): Process: org.jellyfin.androidtv, PID: 18272
11-16 19:38:51.792 E/AndroidRuntime(18272): java.lang.RuntimeException: Unable to start activity ComponentInfo{org.jellyfin.androidtv/org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.List org.jellyfin.sdk.model.api.MediaSourceInfo.getMediaStreams()' on a null object reference
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2914)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3049)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1809)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at android.os.Handler.dispatchMessage(Handler.java:106)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at android.os.Looper.loop(Looper.java:193)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at android.app.ActivityThread.main(ActivityThread.java:6680)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at java.lang.reflect.Method.invoke(Native Method)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
11-16 19:38:51.792 E/AndroidRuntime(18272): Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.List org.jellyfin.sdk.model.api.MediaSourceInfo.getMediaStreams()' on a null object reference
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at org.jellyfin.androidtv.data.compat.StreamInfo.getSubtitleProfiles(StreamInfo.java:301)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at org.jellyfin.androidtv.data.compat.StreamInfo.getSubtitleProfiles(StreamInfo.java:291)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity.adaptExternalSubtitles(ExternalPlayerActivity.java:445)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity.startExternalActivity(ExternalPlayerActivity.java:418)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity.launchExternalPlayer(ExternalPlayerActivity.java:309)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity.onCreate(ExternalPlayerActivity.java:121)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at android.app.Activity.performCreate(Activity.java:7144)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at android.app.Activity.performCreate(Activity.java:7135)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2894)
11-16 19:38:51.792 E/AndroidRuntime(18272): 	... 11 more
11-16 19:38:51.793 E/ACRA    (18272): ACRA caught a RuntimeException for org.jellyfin.androidtv
11-16 19:38:51.793 E/ACRA    (18272): java.lang.RuntimeException: Unable to start activity ComponentInfo{org.jellyfin.androidtv/org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.List org.jellyfin.sdk.model.api.MediaSourceInfo.getMediaStreams()' on a null object reference
11-16 19:38:51.793 E/ACRA    (18272): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2914)
11-16 19:38:51.793 E/ACRA    (18272): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3049)
11-16 19:38:51.793 E/ACRA    (18272): 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
11-16 19:38:51.793 E/ACRA    (18272): 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
11-16 19:38:51.793 E/ACRA    (18272): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
11-16 19:38:51.793 E/ACRA    (18272): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1809)
11-16 19:38:51.793 E/ACRA    (18272): 	at android.os.Handler.dispatchMessage(Handler.java:106)
11-16 19:38:51.793 E/ACRA    (18272): 	at android.os.Looper.loop(Looper.java:193)
11-16 19:38:51.793 E/ACRA    (18272): 	at android.app.ActivityThread.main(ActivityThread.java:6680)
11-16 19:38:51.793 E/ACRA    (18272): 	at java.lang.reflect.Method.invoke(Native Method)
11-16 19:38:51.793 E/ACRA    (18272): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
11-16 19:38:51.793 E/ACRA    (18272): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
11-16 19:38:51.793 E/ACRA    (18272): Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.List org.jellyfin.sdk.model.api.MediaSourceInfo.getMediaStreams()' on a null object reference
11-16 19:38:51.793 E/ACRA    (18272): 	at org.jellyfin.androidtv.data.compat.StreamInfo.getSubtitleProfiles(StreamInfo.java:301)
11-16 19:38:51.793 E/ACRA    (18272): 	at org.jellyfin.androidtv.data.compat.StreamInfo.getSubtitleProfiles(StreamInfo.java:291)
11-16 19:38:51.793 E/ACRA    (18272): 	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity.adaptExternalSubtitles(ExternalPlayerActivity.java:445)
11-16 19:38:51.793 E/ACRA    (18272): 	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity.startExternalActivity(ExternalPlayerActivity.java:418)
11-16 19:38:51.793 E/ACRA    (18272): 	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity.launchExternalPlayer(ExternalPlayerActivity.java:309)
11-16 19:38:51.793 E/ACRA    (18272): 	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity.onCreate(ExternalPlayerActivity.java:121)
11-16 19:38:51.793 E/ACRA    (18272): 	at android.app.Activity.performCreate(Activity.java:7144)
11-16 19:38:51.793 E/ACRA    (18272): 	at android.app.Activity.performCreate(Activity.java:7135)
11-16 19:38:51.793 E/ACRA    (18272): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
11-16 19:38:51.793 E/ACRA    (18272): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2894)
11-16 19:38:51.793 E/ACRA    (18272): 	... 11 more
11-16 19:38:51.793 D/ACRA    (18272): Building report
11-16 19:38:51.795 D/ACRA    (18272): Starting collectors with priority FIRST
11-16 19:38:51.798 D/ACRA    (18272): Calling collector org.acra.collector.StacktraceCollector
11-16 19:38:51.799 D/ACRA    (18272): Calling collector org.acra.collector.DropBoxCollector
11-16 19:38:51.799 D/ACRA    (18272): Collector org.acra.collector.DropBoxCollector completed
11-16 19:38:51.799 D/ACRA    (18272): Calling collector org.acra.collector.LogCatCollector
11-16 19:38:51.799 D/ACRA    (18272): Collector org.acra.collector.StacktraceCollector completed
11-16 19:38:51.805 D/ACRA    (18272): Retrieving logcat output (buffer:default)...

```  

### App information

***App version***: `0.15.0-beta.1` (`150001`)  
***Package name***: `org.jellyfin.androidtv`  
***Build***: 
```json
{"BOARD":"oneday","BOOTLOADER":"unknown","BRAND":"Xiaomi","CPU_ABI":"armeabi-v7a","CPU_ABI2":"armeabi","DEVICE":"oneday","DISPLAY":"PI.3933 release-keys","FINGERPRINT":"Xiaomi\/oneday\/oneday:9\/PI\/3933:user\/release-keys","HARDWARE":"amlogic","HOST":"c5-mitv-bsp-build04.bj","ID":"PI","IS_DEBUGGABLE":false,"IS_EMULATOR":false,"MANUFACTURER":"Xiaomi","MODEL":"MIBOX4","PERMISSIONS_REVIEW_REQUIRED":false,"PRODUCT":"oneday","RADIO":"unknown","SUPPORTED_32_BIT_ABIS":["armeabi-v7a","armeabi"],"SUPPORTED_64_BIT_ABIS":[],"SUPPORTED_ABIS":["armeabi-v7a","armeabi"],"TAGS":"release-keys","TIME":1632823795000,"TYPE":"user","UNKNOWN":"unknown","USER":"jenkins","VERSION":{"ACTIVE_CODENAMES":[],"BASE_OS":"","CODENAME":"REL","FIRST_SDK_INT":28,"INCREMENTAL":"3933","PREVIEW_SDK_INT":0,"RELEASE":"9","RESOURCES_SDK_INT":28,"SDK":"28","SDK_INT":28,"SECURITY_PATCH":"2021-07-05"}}
```  
***Build config***: 
```json
{"APPLICATION_ID":"org.jellyfin.androidtv","BUILD_TYPE":"release","DEBUG":false,"DEVELOPMENT":false,"VERSION_CODE":150001,"VERSION_NAME":"0.15.0-beta.1"}
```  

### Device information

***Android version***: `9`  
***Device brand***: `Xiaomi`  
***Device product***: `oneday`  
***Device model***: `MIBOX4`  

### Crash information

***Start time***: `2022-11-16T19:36:25.592+01:00`  
***Crash time***: `2022-11-16T19:38:51.875+01:00`  



### Application version

0.15.0-beta.1

### Where did you install the app from?

Google Play

### Device information

Xiaomi Mi Box S

### Android version

Android 9

### Jellyfin server version

10.8.7

--- a/app/src/main/java/org/jellyfin/androidtv/data/compat/StreamInfo.java
+++ b/app/src/main/java/org/jellyfin/androidtv/data/compat/StreamInfo.java
@@ -298,6 +298,8 @@ public class StreamInfo {
         long startPositionTicks = getPlayMethod() == PlayMethod.Transcode ? getStartPositionTicks() : 0;
 
         if (!includeSelectedTrackOnly) {
+            if (getMediaSource() == null) return list;
+
             for (org.jellyfin.sdk.model.api.MediaStream stream : getMediaSource().getMediaStreams()) {
                 if (stream.getType() == org.jellyfin.sdk.model.api.MediaStreamType.SUBTITLE) {
                     addSubtitleProfiles(list, stream, enableAllProfiles, baseUrl, accessToken, startPositionTicks);

